cypress/support/step-definitions/common-steps.js

```diff
@@ -31,7 +31,7 @@ Given(
     visitComponentUrlWithParameters(
       `${component}-test`,
       type,
-      "design-system-",
+      "",
```

Comment:
I think we can also delete all that step :) and replace the used step with the step from 42 line

Reply:
This one uses the `test` stories, where the other one does not. I agree this could be simplified but I think it's beyond the scope of this PR.